CHANGELOG.md

```diff
@@ -30,6 +34,8 @@ release.
 
 ### BREAKING
 
+- **Tags**: The gRPC `SetTags` endpoint now prevents setting tags on user-owned nodes. Only nodes registered with tagged PreAuthKeys can have tags modified via the API.
```

Comment:
This is incorrect, a user-owned node can be "converted" to a tagged node by tagging it. That will cause it to become a tagged node and not a user-owned node.

We need to ensure the implementation reflects this.